CONTRIBUTING.md

```diff
@@ -5,7 +5,7 @@ Scio is developed and maintained by an infrastructure team at Spotify. It is the
 
 # Submitting issues
 
-Feel free to discuss issues in the #scio channel on Spotify FOSS Slack (get an invite [here](https://slackin.spotify.com/)) or [Google Group](https://groups.google.com/forum/#!forum/scio-users) first.
+Feel free to discuss issues in the [Google Group](https://groups.google.com/forum/#!forum/scio-users) first.
```

Comment:
We've just enabled GitHub [discussions](https://github.com/spotify/scio/discussions) on scio for that purpose